src/lib/libraries/sprites.json

```diff
@@ -5999,6 +7096,62 @@
             "spriteInfo": {}
         }
     },
+    {
+        "name": "Shark-a",
```

Comment:
@carljbowman I'm guess you meant just "Shark" here.

Reply:
Good catch. Fixed